bin/foreman-maintain

```diff
@@ -4,6 +4,11 @@ $LOAD_PATH.unshift(File.expand_path('../../lib', __FILE__))
 
 require 'foreman_maintain'
 
+CONFIG_FILE = '/etc/foreman-maintain/config/foreman_maintain.yml'.freeze
+STORAGE_DATA_FILE = '/lib/foreman-maintain/data.yml'.freeze
```

Comment:
Isn't needed just the CONFIG_FILE, and we could have different default config files for git and pacakge-based distribution?